README.md

```diff
@@ -112,46 +116,50 @@ FAQ
       * By default, Docker builds can (and often do) run as root. You can control who can create Docker builds through the `builds/docker` and `builds/custom` policy resource.
     * Regular users and project admins cannot change their security quotas.
 
-    Many Docker containers expect to run as root (and therefore edit all the contents of the filesystem). The [Image Author's guide](https://docs.openshift.org/latest/creating_images/guidelines.html#openshift-specific-guidelines) gives recommendations on making your image more secure by default:
+    Many Docker containers expect to run as root, and therefore expect the ability to modify all contents of the filesystem. The [Image Author's guide](https://docs.openshift.org/latest/creating_images/guidelines.html#openshift-specific-guidelines) provides recommendations for making your image more secure by default:
 
-    * Don't run as root
-    * Make directories you want to write to group-writable and owned by group id 0
+    * Don't run as root.
+    * Make directories you want to write to group-writable and owned by group id 0.
     * Set the net-bind capability on your executables if they need to bind to ports &lt;1024
+      (e.g. `setcap cap_net_bind_service=+ep /usr/sbin/httpd`).
 
-    Otherwise, you can see the [security documentation](https://docs.openshift.org/latest/admin_guide/manage_scc.html) for descriptions on how to relax these restrictions.
+    Although we recommend against it for security reasons, it is also possible to relax these restrictions as described in the [security documentation](https://docs.openshift.org/latest/admin_guide/manage_scc.html).
 
 5. How do I get networking working?
 
-    The Origin and Kubernetes network model assigns each pod (group of containers) an IP that is expected to be reachable from all nodes in the cluster. The default setup is through a simple SDN plugin with OVS - this plugin expects the port 4679 to be open between nodes in the cluster. Also, the Origin master processes need to be able to reach pods via the network, so they may require the SDN plugin.
+    The Origin and Kubernetes network model assigns each Pod (group of containers) an IP address that is expected to be reachable from all nodes in the cluster. The default configuration uses Open vSwitch to provide Software-Defined Networking (SDN) capabilities, which requires communication between nodes in the cluster using port 4679.  Additionally, the Origin master processes must be able to reach pods within the network, so they may require the SDN plugin.
 
     Other networking options are available such as Calico, Flannel, Nuage, and Weave. For a non-overlay networking solution, existing networks can be used by assigning a different subnet to each host, and ensuring routing rules deliver packets bound for that subnet to the host it belongs to. This is called [host subnet routing](https://docs.openshift.org/latest/admin_guide/native_container_routing.html).
```

Comment:
@danmcp Wondering if we should add links to the Calico, Flannel, Nuage, and Weave project sites here, or if those options are even recommended for use with OpenShift. Do we provide guides for using those options with OpenShift? If so, maybe we should link to those?